spec.bs

```diff
@@ -1699,6 +1715,44 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>disableUntrustedNetwork()</dfn> method steps are:
+
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=Window/browsing context=]'s
+     [=browsing context/fenced frame config instance=].
+
+  1. If |instance| is null, then return.
```

Comment:
Really these "return"s before creating the promise shouldn't be returns; the API just shouldn't be available in those situations. Not sure how to actually specify that.

Reply:
What do you mean it shouldn't be available? Do we neuter the API surface in the Blink implementation somehow? I would think these should just return rejected promises.

Reply:
I guess I was thinking of window.fence not being available, but for these cases we just return `ScriptPromise<IDLUndefined>()` and log console errors.